docs/3.0.0-beta.x/deployment/amazon-aws.md

```diff
@@ -131,6 +131,8 @@ Follow these steps to complete installation of a `PostgreSQL` database:
     - **Master username:** Keep as `postgres`, or change (optional)
     - `Uncheck` _Auto generate a password_, and then type in a new secret password.
 - **Connectivity**, and **Additional connectivity configuration**: Set `Publicly accessible` to `Yes`.
+- **Security** Edit the existing security group to use a new rule where 
```

Comment:
Two things here:
- Can you please make these changes against the v3 docs, not the v3-beta as we plan to remove them soon as the beta is no longer supported
- Is there an alternative to allowing public internet access to `0.0.0.0/0` as this seems extremely insecure to me. The public web should not be able to query the PG database.